pkg/qemu/qemu.go

```diff
@@ -437,6 +475,10 @@ func Cmdline(cfg Config) (string, []string, error) {
 	} else if !isBaseDiskCDROM {
 		args = append(args, "-drive", fmt.Sprintf("file=%s,if=virtio", baseDisk))
 	}
+	for _, volDisk := range volumeDisks {
+		args = append(args, "-drive", fmt.Sprintf("file=%s,if=virtio", volDisk))
```

Comment:
Needs to add `discard=on` option to enable `fstrim` support. See https://github.com/lima-vm/lima/pull/1102/files.

```suggestion
		args = append(args, "-drive", fmt.Sprintf("file=%s,if=virtio,discard=on", volDisk))
```